subserver_permissions.go

```diff
@@ -163,8 +262,9 @@ func (pm *PermissionsManager) GetLitPerms() map[string][]bakery.Op {
 
 // IsLndURI returns true if the given URI belongs to an RPC of lnd.
 func (pm *PermissionsManager) IsLndURI(uri string) bool {
+	_, lndSubServerCall := pm.lndSubServerPerms[uri]
```

Comment:
This needs to go one level deeper:
```
	lndSubServerCall := false
	for _, subserverPermissions := range pm.lndSubServerPerms {
		_, found := subserverPermissions[uri]
		if found {
			lndSubServerCall = true
			break
		}
	}
```

Reply:
oh wow yeah that is embarrassing 🙈 thanks for catching!